arithmetization/src/main/java/net/consensys/linea/zktracer/ZkTracer.java

```diff
@@ -304,19 +300,24 @@ public void traceContextExit(MessageFrame frame) {
     }
   }
 
-  /** When called, erase all tracing related to the last included transaction. */
-  public void popTransaction(final PendingTransaction pendingTransaction) {
-    if (hashOfLastTransactionTraced.equals(pendingTransaction.getTransaction().getHash())) {
-      hub.popTransaction();
-    }
-  }
-
   private void maybeThrowTracingExceptions() {
     if (!this.tracingExceptions.isEmpty()) {
       throw new TracingExceptions(this.tracingExceptions);
     }
   }
 
+  /**
+   * When called, erase all tracing related to the bundle of all transactions since the last {@link
+   * commitTransactions()}
+   */
+  public void popTransactions() {
+    hub.popTransactions();
+  }
+
+  public void commitTransactions() {
+    hub.commitTransactions();
+  }
+
```

Comment:
Wouldn't it make sense to call it e.g. `popTransactionBundle` / `commitTransactionBundle` or `popBundle` / `commitBundle` ?